.gitignore

```diff
@@ -0,0 +1 @@
+vendor/**
```

Comment:
`/vendor/` to match exactly the directory in the root.

Reply:
PR updated.